.gitignore

```diff
@@ -76,5 +76,3 @@ node_modules/
 crtool/crtool/build/
 crtool/crtool/coverage/
 crtool/static/*
-teachers_digital_platform/static/*
-teachers_digital_platform/tests/unit_test_coverage/
```

Comment:
There's more we could clean up here, but it's probably best to wait until people clean up their environments. 
Wouldn't want to accidentally commit the `.tox` dir.